aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -54,6 +54,25 @@ static void abs_kernel(TensorIterator& iter) {
   });
 }
 
+static void fill_kernel(TensorIterator& iter, Scalar value_scalar) {
+  if( iter.dtype() == ScalarType::Half ) {
+    auto value = value_scalar.to<at::Half>().x;
```

Comment:
Clever. We should probably do this in copy_kernel as well.

Reply:
We might want to rename `x`. Won't work in cases where we want to do any math on the underlying data, but yeah, should work for copy/fill.